src/taipy/core/scenario/_scenario_model.py

```diff
@@ -53,7 +53,7 @@ class _ScenarioModel(_BaseModel):
     subscribers: List[Dict]
     tags: List[str]
     version: str
-    pipelines: Optional[List[PipelineId]] = None
+    pipelines: Optional[Union[Dict[str, PipelineId], Dict]] = None
```

Comment:
This type hint seems weird.
Can it be `Optional[Dict[str, PipelineId]]`?
Or `Optional[Union[Dict[str, PipelineId], Dict[str, Pipeline]]]`